fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2474,6 +2476,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static String metadata_journal_skip_bad_journal_ids = "";
 
+    /**
+     * Set this configuration to true to ignore specific operation (with IgnorableOnReplayFailed annotation) replay failures.
+     */
+    @ConfField
+    public static boolean ignore_journal_replay_failure = false;
```

Comment:
For newly added configuration, we should make it the same prefix, like `journal_replay_ignore_failure`, `journal_replay_skip_bad_ids`, etc.

Reply:
Change it to `metadata_journal_ignore_replay_failure`, same prefix to `metadata_journal_skip_bad_journal_ids`